README.md

```diff
@@ -24,13 +24,13 @@ _embedded_-_friendly_ adaptation of the aforementioned work.
 The `avr-libstdcpp` port began in 2018 with an initial import
 of a GNU/GCC-based C++ standard library from GCC 8.
 A second import of a GNU/GCC-based C++ standard library from GCC 10
-in 2020 modernized the port to include many modern C++20 features.
+in 2020 modernized the port to include many contemporary C++20 features.
 
 ## Using the library
 
-- Add the `avr-libstdcpp/include` path to the standard `-I` include path(s) of the compiler on the command line.
+- Add the `avr-libstdcpp/include` path to the standard `-isystem` (or `-I`) include path(s) of the compiler on the command line.
 - Upon doing this, include standard library headers in the usual way (i.e., `#include <algorithm>`,  `#include <array>`,  `#include <cstdint>`, etc.).
-- There are also a handful of source files located in the [src directory](./src). Some of these may potentially be needed.
+- There is also a handful of source files located in the [src directory](./src). Some of these may potentially be needed.
```

Comment:
I think the original is correct here. "There are a handful" is plural, like "there are a couple" or "there are a few".

Reply:
Iw asn't the greatest wording in the first place. Since we mean _there_ _are_ _files_, let's simplify it.

"There are also several source files located in ..."